packages/termux-am-2/build.sh

```diff
@@ -0,0 +1,11 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/tareksander/termux-am-2
```

Comment:
Maybe termux-am-socket, or similar, could be a more descriptive name for the repo and package?

Reply:
I feel, the repo can be transferred to `@termux`. It will make maintaince of the utility easier for us.